siac/hostcmd.go

```diff
@@ -358,7 +358,7 @@ func hostannouncecmd(cmd *cobra.Command, args []string) {
 	case 1:
 		err = post("/host/announce", "netaddress="+args[0])
 	default:
-		cmd.Usage()
+		cmd.UsageFunc()
```

Comment:
What is the difference? I ran 'siac' to make sure that the usage output was correct